### DISCUSSION TITLE
Educational benefits

### DISCUSSION THREAD

**Discussion by ByteBelle-21** [community/community#145341] [GitHub Education]
### Hi everyone,

Hello, 
I have been approved to get educational benefits, but when the benefits will be available ?

**Comment by OmgRod** [comment]
They should be available in 1-3 days but for me it took 4 days.

**Reply by OmgRod** [reply]
tl;dr it's unpredictable

**Comment by davevad93** [comment]
Hi @ByteBelle-21 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ByteBelle-21 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Aygyun951** [comment]
@ByteBelle-21  Hello, Did you get yours yet? I am currently in the same situation.
